dist/components/Node.js

```diff
@@ -56,11 +56,11 @@ var Node = function (_React$Component) {
   (0, _createClass3.default)(Node, [{
     key: 'render',
     value: function render() {
-      var _props = this.props;
-      var node = _props.node;
-      var depth = _props.depth;
-      var tagStyle = stylesheet.tagStyle;
-      var containerStyle = stylesheet.containerStyle;
+      var _props = this.props,
+          node = _props.node,
```

Comment:
I've always liked single line declarations over multiline ones. I believe there is an [eslint rule](https://github.com/airbnb/javascript/blob/772bbb5b7d2f6990e519c3d70539f807257492fe/packages/eslint-config-airbnb-base/rules/style.js#L311) to suggest that too. I've not run this changelist for eslint but IMO this change is not needed.